efsity-cli/src/main/java/org/smartregister/command/QuestionnaireResponseGeneratorCommand.java

```diff
@@ -378,12 +383,17 @@ static String populateMode(String questionnaireData, String fhir_base_url, Strin
     if (questionnaire_response.has("contained")) {
       questionnaire_response.remove("contained");
     }
-    JSONArray response = (JSONArray) questionnaire_response.get("item");
-    JSONArray questions = resource.getJSONArray("item");
-    FctUtils.printInfo("Debug: questionnaire_response before line 379: " + questionnaire_response);
-
-    JSONArray response_with_answers = getAnswers(questions, response, extras);
-    questionnaire_response.put("item", response_with_answers);
+    if (questionnaire_response.has("item")) {
+      JSONArray response = (JSONArray) questionnaire_response.get("item");
+      JSONArray questions = resource.getJSONArray("item");
+      FctUtils.printInfo(
+          "Debug: questionnaire_response before line 379: " + questionnaire_response);
```

Comment:
Can we maybe have this message be something more generic? Like `FctUtils.printError ("Debug: response from questionnaireResponse generation: ...")` 
I don't think it's obvious what "before line 379" means to everyone who would potentially be running this